tests/Unit/Service/CartLinesServiceTest.php

```diff
@@ -0,0 +1,363 @@
+<?php
+
+namespace Service;
+
+use Module;
+use Mollie\Adapter\ConfigurationAdapter;
+use Mollie\Config\Config;
+use Mollie\DTO\Line;
+use Mollie\DTO\Object\Amount;
+use Mollie\Repository\AttributeRepository;
+use Mollie\Service\CartLinesService;
+use Mollie\Service\VoucherService;
+use PHPUnit\Framework\MockObject\MockObject;
+use PHPUnit\Framework\TestCase;
+
+class CartLinesServiceTest extends TestCase
+{
+
+    /**
+     * @dataProvider cartLinesProvider
+     *
+     * @param $amount
+     * @param $paymentFee
+     * @param $currencyIsoCode
+     * @param $cartSummary
+     * @param $shippingCost
+     * @param $cartItems
+     * @param $psGiftWrapping
+     * @param $selectedVoucherCategory
+     * @param $mocks
+     * @param $result
+     */
+    public function testGetCartLines(
+        $amount,
+        $paymentFee,
+        $currencyIsoCode,
+        $cartSummary,
+        $shippingCost,
+        $cartItems,
+        $psGiftWrapping,
+        $selectedVoucherCategory,
+        $mocks,
+        $result
+    ) {
+        $mollieModule = Module::getInstanceByName('mollie');
```

Comment:
for the reference , next time dont initialize whole Module - even better you dont need to use whole module in services at all in all cases